README.md

```diff
@@ -15,21 +19,53 @@ The artefact mainly consists of a library written in the Haskell language that i
 The library is accompanied with a demo application that uses the library to reproduce our motivating example (Alice and Bob using feature trace recording in Section 2 in our paper) as well as examples of the edit patterns we used to evaluate feature trace recording (Section 5).
 
 
+## What is Feature Trace Recording?
+
+Feature trace recording is a semi-automated method for recording feature-to-code mappings during software development.
+It unlocks evolution as a new source of information for feature location.
+
+![Feature Trace Recording Example](meta/FeatureTraceRecording.png)
```

Comment:
The image is rendered really badly for me. It is kind of blurry and hurts the eyes. Strangely, everything appears to be fine when opening the image. 
![Screenshot_20210716_145059](https://user-images.githubusercontent.com/54794966/125950503-b2563b08-0ba7-4e18-a2d2-fc838daeb438.png)

Reply:
So this seems to be an issue with the markdown viewer you use. Which one was it? It looks fine for me in VSCode and on GitHub.

Reply:
Also I have no idea how to change that as the image is fine as you said.

Reply:
It was on GitHub using Google Chrome

Reply:
Could be fixed by creating custom HTML to integrate the image... I have no idea how though and it that would work. Sounds like a general bug in Github+Chrome. :(

Reply:
I tried a fix, please recheck.

Reply:
ok fixed